.travis.yml

```diff
@@ -7,10 +7,17 @@ php:
 env:
   - MONGO_VERSION=1.2.12
   - MONGO_VERSION=1.3.3
+  - SYMFONY_VERSION=2.1.*
+  - SYMFONY_VERSION=2.2.*
+  - SYMFONY_VERSION=2.3.*
+  - SYMFONY_VERSION=2.4.*
+  - SYMFONY_VERSION=2.5.*
+  - SYMFONY_VERSION=2.6.*
+  - SYMFONY_VERSION=2.7.*
```

Comment:
Please revert this change:

- it is the wrong way to do this as you make the build matrix explode (especially once adding more PHP versions)
- you are not testing against different Symfony version, only against different FrameworkBundle version (but you are still getting 2.6 for many components, including DependencyInjection)

The right way is to test againts the lowest deps, which is something I'm currently doing and will submit separately